restaurant-front/src/modules/shared/models/Drink2.ts

```diff
@@ -0,0 +1,10 @@
+import { Ingredient } from "./ingredient";
+
+export interface Drink2{
```

Comment:
Rename (maybe AddDrinkWithIngredients / AddBartenderDrink)

Reply:
Can it be BartenderDrink, because it is not only in adding used?